style.css

```
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Bantangas";
+  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff");
+  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff2");
```

Switch these two, so woff2 is used by browser if possible (newer).
```suggestion
  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff2");
  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff");
  ;
```